src/layouts/Media/components/MediaBreadcrumb.tsx

```diff
@@ -45,21 +50,34 @@ export const MediaBreadcrumbs = (): JSX.Element => {
           ]
         )
         .map(({ name, url }, idx) => {
-          const hasModifier = idx === directories.length - 1
+          if (
+            directories.length > MAX_MEDIA_LEVELS &&
+            idx > 1 &&
+            idx < directories.length - (MAX_MEDIA_LEVELS - 1)
```

Comment:
can add comment?

Reply:
Added and adjusted in 88205a2e! I realised it's a different scenario because even if we increase the MAX_MEDIA_LEVELS we do not want to display more than 3 albums/directories.